src/screens/Messages/Conversation/index.tsx

```diff
@@ -154,12 +142,19 @@ let Header = ({
       )}
       <View style={[a.align_center, a.gap_sm, a.flex_1]}>
         {profile ? (
-          <>
+          <View style={[a.align_center]}>
```

Comment:
This whole header is getting a little cramped, we may want to consider doing a horizontal layout so it doesn't get too tall